test/features/articles/deleting_an_article_test.rb

```diff
@@ -1,12 +1,36 @@
 require "test_helper"
 
 feature "Deleting an article" do
-  scenario "is successful when user clicks destroy link" do
-    article = articles(:two)
+  before do
+    @article = articles(:one)
+    @article.save
+  end
+
+  scenario "is successful when editor clicks destroy link" do
+    sign_in(:editor)
+    article = articles(:one)
     visit article_path(article)
-    find('body').must_have_content "Second Post"
+    find('body').must_have_content "First Post"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.